crates/trie/common/benches/prefix_set.rs

```diff
@@ -115,8 +115,13 @@ fn prefix_set_bench<T>(
 fn generate_test_data(size: usize) -> (Vec<Nibbles>, Vec<Nibbles>, Vec<bool>) {
     use prop::collection::vec;
 
-    let config = ProptestConfig { result_cache: basic_result_cache, ..Default::default() };
-    let mut runner = TestRunner::new(config);
+    let config = ProptestConfig {
+        result_cache: basic_result_cache,
+        rng_algorithm: RngAlgorithm::ChaCha,
+        ..Default::default()
+    };
+    let mut runner =
+        TestRunner::new_with_rng(config, TestRng::deterministic_rng(RngAlgorithm::ChaCha));
```

Comment:
would be nice if we could do this
```suggestion
    let config = ProptestConfig {
        result_cache: basic_result_cache,
        ..Default::default()
    };
    let mut runner =
        TestRunner::new_with_rng(config, TestRng::deterministic_rng(config.rng_algorithm));
```